pkgs/development/python-modules/wxPython/4.0.nix

```diff
@@ -10,6 +10,9 @@
 , doxygen
 , ncurses
 , wxGTK
+, wxmac
+, darwin
```

Comment:
We usually don't import `darwin` as a whole and provide frameworks one by one.

Reply:
@fricklerhandwerk please address this.

Reply:
Thanks for the heads up, the comment seems to have slipped by me, sorry. Will fix this week.